app/src/main/java/ru/firstSet/kotlinOne/DataSource/MoviesDataSource.kt

```diff
@@ -0,0 +1,133 @@
+package ru.firstSet.kotlinOne.DataSource
+
+import ru.firstSet.kotlinOne.Data.Movie
+import ru.firstSet.kotlinOne.R
+
+class MoviesDataSource {
```

Comment:
@kamar19 Please use `object` to make this class singleton

Reply:
Fixed it. Read again about singleton class in Kotlin